Formula/s/smali.rb

```diff
@@ -19,6 +19,8 @@ class Smali < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "a674a05a86c284a8f15d82c01e73af0e1802b755659c6417d79157ff2108f82f"
   end
 
+  deprecate! date: "2023-12-14", because: "uses deprecated `gradle@6` to build"
```

Comment:
Could use bundled `gradlew` but that would end up using its own copy of Gradle 6.

At least Google's fork considers the project unmaintained: https://github.com/google/smali#about

Reply:
just a thought, maybe move to the google fork?

Reply:
Assuming they are using `gradlew` then it is still building with Gradle 6.

Reply:
make sense 👍